utils.js

```diff
@@ -0,0 +1,111 @@
+"use strict";
+
+const IOS_PLATFORM = "iOS";
+const ANDROID_PLATFORM = "Mobile";
+const iOSFixedFirefoxVersion = "29.0";
+
+const DESKTOP_NIGHTLY_KEY = "FIREFOX_NIGHTLY";
+const MOBILE_NIGHTLY_KEY = "nightly_version";
+
+const PRODUCT_ENDPOINT = "https://product-details.mozilla.org/1.0/";
+const DESKTOP_ENDPOINT = `${PRODUCT_ENDPOINT}firefox_versions.json`;
+const MOBILE_ENDPOINT = `${PRODUCT_ENDPOINT}mobile_versions.json`;
+
+const WINDOWS = "Windows 10";
+const iOS = "iOS 14.1";
+const macOS = "Mac OS X 10.15";
+
+const OS_REPLACE = {
+  All: WINDOWS,
+  Windows: WINDOWS,
+  macOS: macOS,
+  iOS: iOS
+};
+
+function getOS(os) {
+  if (!os) {
+    return WINDOWS;
+  }
+
+  if (os in OS_REPLACE) {
+    return OS_REPLACE[os];
+  }
+
+  return os;
+}
+
+function getPlatform(os) {
+  if (os.includes("Android")) return ANDROID_PLATFORM;
+  else if (os.includes("iOS")) return IOS_PLATFORM;
+
+  return "";
+}
+
+/**
+ * Returns a string containing version in NN.N format
+ *
+ * @param n
+ * @param os
+ * @returns {string}
+ */
+
+function getVersionNum(n, os) {
+  if (os.includes("iOS")) return iOSFixedFirefoxVersion;
```

Comment:
I've hardcoded this as I'm not sure where to get recent Firefox iOS version from. This api only has version of OS https://product-details.mozilla.org/1.0/mobile_versions.json

Reply:
Could you file an issue to investigate that eventually? :)

Reply:
Yeah, filed https://github.com/webcompat/bugzilla-to-webcompat/issues/8